test/functional/functest.js

```diff
@@ -167,9 +176,3 @@ describe('FuncTest', function(done) {
         });
     })
 });
-
-// The tests with ISA require an additional library. They are disabled for now because of this.
```

Comment:
If you added the backend, doesn't it make sense to activate them instead?

In any case, some tests would be welcome here.